examples/bpi-monolith-test-suite/bpi/bpi.ts

```diff
@@ -0,0 +1,136 @@
+import { Agreement } from "./components/storage/agreement";
+import { BpiSubject } from "./components/identity/bpiSubject";
+import { Invitation } from "./components/workgroup/invitation";
+import { Workgroup } from "./components/workgroup/workgroup";
+import { Workstep } from "./components/workgroup/workstep";
+import { BpiMessage } from "./components/messaging/bpiMessage";
+import { TransactionPoolComponent } from "./components/transactions/transactionPool";
+import { IMessagingComponent } from "./components/messaging/messaging.interface";
+import { IWorkgroupComponent } from "./components/workgroup/workgroup.interface";
+import { IIdentityComponent } from "./components/identity/identity.interface";
+import { ITransactionPoolComponent } from "./components/transactions/transactionPool.interface";
+import { IVirtualStateMachineComponent } from "./components/vsm/vsm.interface";
+import { VirtualStateMachine } from "./components/vsm/vsm";
+import { IStorageComponent } from "./components/storage/storage.interface";
+import { StorageComponent } from "./components/storage/storage";
+
+export class BPI {
+    identityComponent: IIdentityComponent;
+    messagingComponent: IMessagingComponent;
+    workgroupComponent: IWorkgroupComponent;
+    transactionPoolComponent: ITransactionPoolComponent;
+    vsmComponent: IVirtualStateMachineComponent;
+    storageComponent: IStorageComponent;
+
+    constructor(
+        id: string,
+        name: string,
+        productIds: string[],
+        identityComponent: IIdentityComponent,
+        messagingComponent: IMessagingComponent,
+        workgroupComponent: IWorkgroupComponent) {
+
+        this.storageComponent = new StorageComponent();
+        const initalAgreement = new Agreement();
+        initalAgreement.productIds = productIds;
+        this.storageComponent.setInitialAgreementState(initalAgreement);
+
+        this.identityComponent = identityComponent;
+        const ownerOrg = this.identityComponent.addOrganization(id, name);
```

Comment:
for standard compliance, this should be `bpiSubjectOwner` and the exported module should be `addBPISubject`. @ognjenkurtic

Reply:
will change as part of [these](https://github.com/eea-oasis/baseline/projects/2)